src/core_plugins/testbed/public/testbed.html

```diff
@@ -5,6 +5,10 @@
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
 
+    <div class="kuiControlledTable">
+      <loading-results results-name="all the things"></loading-results>
```

Comment:
So, just so I'm clear, I'm going to do the following as part of this PR:
* rename this directive from `loadingResults` to `tableInfo`, 
* rename the `kuiLoadingItems` class in the UI Framework to `kuiTableInfo`,
* get rid of the `kuiNoItems` class in the UI Framework,
* update all usages affected by the above changes